src/components/Footer.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
```

Comment:
dobrą praktyką jest sortowanie importów w kolejności:

1. packi npm (import styled from 'styled-components') - jeśli korzystamy z Reacta i musimy go zaimportować (od którejś wersji już tego robić nie trzeba) to ten import będzie pierwszy
2. ścieżki absolutne (import myFunc from 'src/utils/myFunc')
3. ścieżki relatywne (import myFunc from '../../myFunc)